characters/benjamin.md

```diff
@@ -2,14 +2,20 @@
 
 ## Appearance
 
-Benjamin is a 7' tall white human with black hair and blue eyes.
-He weights 270 (inches? your character sheet says inches man.).
-At 35, his hair is speckled with gray (or is he bald?), but he still has much of the vigor a young man would.
+Benjamin is a 6'2" tall, white human with black hair and blue eyes.
+He weights 215 pounds.
+At 35, his hair is speckled with gray, but he still has much of the vigor of a younger man.
 
 ## Background
 
-Benjamin was born to a single mother, whom he is (was?) exceptionally protective of.
-He spent the better part of his life serving in the Themain city guard, honing his skills while protecting the law.
+Benjamin was born to a single mother - Cassandra Wark - of whom he was exceptionally protective.
+He spent his adolescence and early adulthood serving in the Themain city guard, honing his skills while protecting the law -- all in pursuit of providing for his poor, feeble and lower-class mother.  His father, never known to him, was rarely spoken of and only ever in hushed, dismissive tones -- though Ben always believed him a heroic figure, bold and stalwart.  All he knew for sure was that he left suddenly in the night, some time before his birth.  It must have been for a good reason, he assured himself.
+
+When Ben was 25, while serving a watch shift at dusk, lowlife criminals burglerized his home to take what little valuables he and Cassandra had squirrled away over the years for their own.  Attempting to defend herself and the wealth her dear son had labored to accumulate, Ben's mother foolishly pulled a knife on the intruders.  She lost the fight, and the scene Benjamin returned to at dawn has haunted him ever since.
+
+After that, Benjamin joined the military proper, valuing his life very little -- but finding purpose in protecting the innocent, especially women and children.  His demeanor should come across as rude and presumtuous; but always in an effort keep those who he assumes weak away from harm.  He must die first.
```

Comment:
"His demeanor should come" -- can we change it to "His demeanor comes across as..."?